tests/circuit_graph/test_qasm.py

```diff
@@ -349,9 +341,8 @@ def qnode():
             return qml.expval(qml.PauliZ(0))
 
         # construct the qnode circuit
-        qnode()
-
-        res = qnode.qtape.to_openqasm()
+        tape = qml.workflow.construct_tape(qnode)()
```

Comment:
```suggestion
        tape = qml.tape.QuantumScript([], [qml.expval(qml.Z(0))])
```

Might be bigger pain to make these changes, but did we really need a qnode and device at all?

This will make it easier to make other changes in the future.

Reply:
Yes, good point. I will update that!

Reply:
Addressed in latest commit.